common/src/main/res/values/strings.xml

```diff
@@ -769,4 +769,6 @@
     <string name="tile_icon">Select a icon for the tile</string>
     <string name="tile_select">Select a tile to edit</string>
     <string name="shortcut_pinned">Pinned Shortcuts</string>
+    <string name="remote_debugging">Remote Debugging</string>
+    <string name="remote_debugging_summary">Allow remote debugging to troubleshoot Home Assistant frontend issues</string>
```

Comment:
I would mention that it's just for the Webview rather than just `Remote Debugging` because that might cause people to think they can debug more than just the webview.

Reply:
Ok I'll add that, I tried to avoid that terminology to not confuse users but in this case it's specifically for the webview so that makes sense